client/src/components/vote/ResultsModal.tsx

```diff
@@ -15,39 +16,89 @@ export default class ResultsModal extends Component<IResultsModalProps> {
     super(props);
   }
 
+  public getSortedAnswersWithResults = () => {
+    const results = this.props.results.map((result) => parseInt(result, 10));
+    // const resultsPercentage = this.props.results.map((result) => parseInt(result, 10)/results);
+
+    results.sort().reverse();
+    const votingInfo: VotingInfo = this.props.voting.info;
+    const dic = [];
```

Comment:
```suggestion
    const dic: Array<{ answer: string; result: number }> = [];
```
add function return type (implicitly)

Reply:
It's easier to write `map/reduce` functions